scripts/create_testenv.sh

```diff
@@ -86,3 +87,7 @@ fi
 #  Install editable using the setup.py
 pip install  --no-cache-dir -r requirements.txt
 pip install --no-cache-dir -r requirements-dev.txt
+
+if [ "$NAME" = "SPHINX" ]; then
+    conda install -y -c conda-forge selenium phantomjs
```

Comment:
At this point we might want to split requirements into `requirements-dev,txt` and `requirements-docs.txt` so people can build docs locally. No need to do on this pr if you dont want to. Just something to think about

Reply:
I thought about this, but only selenuim can be pip installed, phantomjs has to be installed with conda or npm